flink-python/setup.py

```diff
@@ -108,8 +119,13 @@
             print(incorrect_invocation_message, file=sys.stderr)
             sys.exit(-1)
 
-        if getattr(os, "symlink", None) is not None:
+        try:
```

Comment:
What's the reason of this change? I believe there must be some reason that I don't know.

Reply:
There are cases the os support symlink however the disk doesn't support symlink. This is found when we test this PR on a windows virtual machine in which the disk is a mounted virtual disk.